app/js/views/main.js

```diff
@@ -151,14 +204,75 @@ export default class MainView extends View {
     return mainViewTemplate;
   }
 
-  render() {
-    super();
-
-    this.el.appendChild(this.actionMenuView.el);
+  _addChildViews() {
```

Comment:
I tried something like that, and had trouble getting the animation right. I could try again. A better approach might actually be to not even create those views until we open up, so we can keep the customizer really light-weight when it is not in use.  Again, though, that's out of scope for this bug.